contracts/nois-proxy/src/state.rs

```diff
@@ -33,6 +33,8 @@ pub struct Config {
     /// The prices of a randomness. List is to be interpreted as oneof,
     /// i.e. payment must be paid in one of those denominations.
     pub prices: Vec<Coin>,
+    /// Manager to set the config
+    pub manager: Addr,
```

Comment:
Can you make that optional for cases lile Stargaze?

```suggestion
    /// Manager to set the config and withdraw funds. If no manager is set, the functionality is not available.
    pub manager: Option<Addr>,
```